### PR TITLE
Improved Dashboard popups to MaterialUI

### PR DESCRIPTION
**Pull request by theprover97**

### CODE REVIEW

**Comment by theprover97** [comment]
NewEvent

**Comment by theprover97** [comment]
NewProject

**Comment by Rupeshiya** [comment]
cc @jaskirat2000 
Give your feedback please

File: src/user/dashboard/popups/CreateProject.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import React, {Component} from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import {Dialog, DialogContent, DialogTitle, Button, Grid} from '@material-ui/core';
+import {TextField, InputLabel} from '@material-ui/core';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import "../../profile/popups/popups.scss";
+
+export class CreateProject extends Component{
+    // eslint-disable-next-line
```
Comment (Rupeshiya) [inline src/user/dashboard/popups/CreateProject.js]: enable the eslint, please.

File: src/user/profile/popups/popups.scss [hunk 2]
```diff
@@ -1,100 +1,139 @@
...
-    font-weight: 500;
+    font-weight: 600;
     font-size: 10px;
     line-height: 14px;
     letter-spacing: 0.2em;
     color: #90949C;
     margin-top:10px;
 }
 .message{
+    font-family: Inter;
```
Comment (Rupeshiya) [inline src/user/profile/popups/popups.scss]: Make sure you use same font as in other popups.

File: src/user/dashboard/popups/CreateEvent.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
+import React, {Component} from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import {Dialog, DialogContent, DialogTitle, Button, Grid} from '@material-ui/core';
+import {TextField, InputLabel} from '@material-ui/core';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import "../../profile/popups/popups.scss";
+
+export class CreateEvent extends Component{
+    // eslint-disable-next-line
```
Comment (Rupeshiya) [inline src/user/dashboard/popups/CreateEvent.js]: .

File: src/user/dashboard/news-feed/news-feed.js [hunk 4]
```diff
@@ -173,21 +168,56 @@ class NewsFeed extends Component {
...
+    );
+  }
+}
+
+
+export default NewsFeed;
+
+/*
+<Modal show={this.state.show} onHide={this.handleClose} animation={false} className="modal-box">
                 <Modal.Header closeButton className="modal-header" style={borderStyle}>
```
Comment (Rupeshiya) [inline src/user/dashboard/news-feed/news-feed.js]: Remove commented code.

**Comment by jaskirat2000** [comment]
Taking into consideration bit similarity the "No" button in second image is not same as the first one. 
Rest seems fine to me.

**Comment by theprover97** [comment]
@jaskirat2000 the "No" button in 2nd image was under 'hover' at time of screenshot. Both have same shadow.

**Comment by Rupeshiya** [comment]
@theprover97 Please squash your commits.

**Comment by theprover97** [comment]
@Rupeshiya did you use material-ui version ^4.8.0. Because the code is compiling without error in my machine.

**Comment by Rupeshiya** [comment]
@theprover97 Please check the package.json file is not updated with the material ui-package, check in PR. I think you have installed the package globally and then running, please update the package.json file with the material-ui package.
Hope you get it.

**Comment by theprover97** [comment]
@Rupeshiya I have updated package.json. Please check. Thanks for your review.

**Comment by jaskirat2000** [comment]
Great work. Are these only few pop ups. Right?

**Comment by Rupeshiya** [comment]
yes, @jaskirat2000 As of now in the dashboard we have only these popups.
@theprover97 Please squash the commits.

**Comment by jaskirat2000** [comment]
These are just few pop ups. Seems like we should complete the whole dashboard

**Comment by Rupeshiya** [comment]
yes @jaskirat2000 You are right, there are more popups to be added as per figma design and enhance the dashboard.

**Comment by jaskirat2000** [comment]
Yeah

**Comment by theprover97** [comment]
@jaskirat2000 @Rupeshiya there is another popup in Dashboard, named 'New Post'. But, this popup only includes a 'Post Description', which can already be filled at the 'Write a Post' input field.

**Comment by theprover97** [comment]
I am thinking of adding following behaviour to this. On clicking the 'Write a Post' field, a popup opens with fields of 'Post Title' and 'Post Description'. On clicking 'Save', the popup closes, and the 'Post Title' fills the 'Write a Post' field. At this time, a button appears along with this labelled as 'Post'. Clicking it marks submitting your post, and the button disappears, along with the text in the 'Write Post' field.

**Comment by theprover97** [comment]
@jaskirat2000 @Rupeshiya please provide your reviews on this.

**Comment by jaskirat2000** [comment]
Umm. Sounds good to me

**Comment by Rupeshiya** [comment]
@theprover97 you can check the schema of the post in backend repo and make the fields according to that.
There will be description, image etc..
Please check that once.

**Comment by theprover97** [comment]
Sure @Rupeshiya , I will check that to improve the design.

**Comment by Rupeshiya** [comment]
@theprover97 Whats the update on this?

**Comment by theprover97** [comment]
@Rupeshiya I have made the component, just completing the behavior as explained above.

**Comment by theprover97** [comment]
@Rupeshiya @jaskirat2000 I have added the Create Post popup. Please check!

**Comment by theprover97** [comment]
Screenshot from 2020-04-02 12-14-26

**Comment by theprover97** [comment]
Screenshot from 2020-04-02 12-14-49

**Pull request merged**